app/src/main/kotlin/com/wire/android/ui/home/conversationslist/AllConversationScreen.kt

```diff
@@ -56,8 +64,10 @@ private fun AllConversationContent(
                 ConversationItemFactory(
                     conversation = conversationItem,
                     eventType = eventType,
-                    onConversationItemClick = onConversationItemClick,
-                    onConversationItemLongClick = onEditConversationItem
+                    openConversation = { onOpenConversation(conversationItem.id) },
+                    onConversationItemLongClick = { onEditConversation(conversationItem.conversationType) },
+                    openUserProfile = onOpenUserProfile,
+                    onMutedIconClick = { openConversationNotificationsSettings(conversationItem.conversationType) },
```

Comment:
not 100 % sure, but I see a optential problem here with recomposition, as we are not really focusing on making MVP as fast as possible for now :P do not know if it is a big of an issue but : 

because we are creating a lambda here each time inside ConversationItemFactory() it will get recomposed even though converstaionItem or eventType will not change

Reply:
Fixed